postmark/django_backend.py

```diff
@@ -82,7 +82,7 @@ def _build_message(self, message):
                     html_body = alt[0]
                     break
 
-        elif getattr(message, 'content_subtype', None) == 'html':
+        if getattr(message, 'content_subtype', None) == 'html':
```

Comment:
this is the only substantial change.  The rest is just django 2 wiring and tests